### ISSUE TITLE
show all results of FASTQC analysis in FASTQC viewer and allow user to scroll through

### ISSUE THREAD

**Issue by ngehlenborg** [refinery-platform/refinery-platform#659]
User reported issue:

> it would be nice if we could scroll and see the entire FastQC visualization. It would also be great to compare two+ side-by-side.

**Comment by scottx611x** [comment]
May not be relevant after #2042

**Comment by jkmarx** [comment]
@scottx611x I second this issue is no longer relevant.
